### PR TITLE
Parties and parliament members

### PR DESCRIPTION
**Pull request by AriHrannar**

### CODE REVIEW

File: district/tests.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (Jarngreipr) [inline district/tests.py]: I would really like to see a complete test suite here before this is merged.

File: parliament/migrations/0002_parliamentmember.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-04-25 17:58
+from __future__ import unicode_literals
```
Comment (Jarngreipr) [inline parliament/migrations/0002_parliamentmember.py]: This is not applicable since we are not running on python 2

File: party/apps.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from __future__ import unicode_literals
```
Comment (Jarngreipr) [inline party/apps.py]: Not applicable to python 3

File: party/migrations/0001_initial.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-04-25 17:58
+from __future__ import unicode_literals
```
Comment (Jarngreipr) [inline party/migrations/0001_initial.py]: Python 2 code

File: party/models.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+from __future__ import unicode_literals
```
Comment (Jarngreipr) [inline party/models.py]: suggest this be removed :)

File: parliament/migrations/0002_parliamentmember.py (hunk 2 above)
Comment (AriHrannar) [inline parliament/migrations/0002_parliamentmember.py]: This is auto generated by django, ill create an issue and someone can look into it

Comment (AriHrannar) [inline parliament/migrations/0002_parliamentmember.py]: #12

File: party/migrations/0001_initial.py (hunk 4 above)
Comment (AriHrannar) [inline party/migrations/0001_initial.py]: #12

File: parliament/migrations/0002_parliamentmember.py (hunk 2 above)
Comment (Ragnar-Hardarson) [inline parliament/migrations/0002_parliamentmember.py]: Migrations are auto-generated files and should not be touched manually.

That being said. IMO, removing `-*- coding: utf-8 -*-` manually is fine, but not required for this PR.

File: party/tests.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
+from rest_framework.test import APITestCase
+from rest_framework.test import APIClient
+from .models import Party
+
+
+class PartyAPI(APITestCase):
+
+    def setUp(self):
+        p = Party(name='Party', website='www.website.com')
```
Comment (Ragnar-Hardarson) [inline party/tests.py]: Nitpicking
Could you update this to `name='Party_name` or `name=`Framsókn`

Preferably "Framsókn" as we'll be dealing with special characters sooner or later

File: politech_backend/settings.py [hunk 7]
```diff
@@ -45,6 +45,8 @@
     'rest_framework.authtoken',
     'main',
     'parliament',
+    'party',
```
Comment (Ragnar-Hardarson) [inline politech_backend/settings.py]: Nitpicking:
Could you sort the custom developed ones alphabetically?
Makes it easier to add apps in the future

**Pull request merged**